base/tekton.dev/tasks/build-layer.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: build-layer
+spec:
+  steps:
+    - image: quay.io/podman/stable:latest
+      name: build-layer
+      resources: {}
+      securityContext:
+        privileged: true
+      script: |
+        #!/usr/bin/env bash
```

Comment:
Could this Task be separated in future and run in a separate pipeline (like the 'future' payload one, as we can do for okd/fcos) and maintain the coreos pipeline to only build the base os?